### PR TITLE
Added error logging to Company.downloaders

### PR DESCRIPTION
**Pull request by IegorSoboliev**

### CODE REVIEW

File: business_register/converter/company_converters/uk_company.py [hunk 1]
```diff
@@ -119,12 +119,16 @@ def update(self):
...
+        try:
+            UkCompanyConverter().save_to_db(self.file_path)
+            self.report.update_finish = timezone.now()
+            self.report.update_status = True
+            self.report.save()
+            logger.info(f'{self.reg_name}: save_to_db({self.file_path}) finished successfully.')
+        except Exception as e:
+            self.report.update_finish = timezone.now()
+            self.report.update_message = e
+            self.report.save()
```
Comment (roman-tiukh) [inline business_register/converter/company_converters/uk_company.py:131]: raise?

File: business_register/converter/company_converters/ukr_company.py [hunk 2]
```diff
@@ -790,13 +790,16 @@ def update(self):
...
+        try:
+            ukr_company.process()
+            self.report.update_finish = timezone.now()
+            self.report.update_status = True
+            self.report.save()
+            logger.info(f'{self.reg_name}: process() with {self.file_path} finished successfully.')
+        except Exception as e:
+            self.report.update_finish = timezone.now()
+            self.report.update_message = e
+            self.report.save()
```
Comment (roman-tiukh) [inline business_register/converter/company_converters/ukr_company.py:802]: raise?

**Review by roman-tiukh — COMMENTED** [review]
PR contains duplicated logic

File: business_register/converter/company_converters/uk_company.py (hunk 1 above)
Comment (IegorSoboliev) [reply business_register/converter/company_converters/uk_company.py:131]: We have to remove the file and measure changes)

Comment (IegorSoboliev) [reply business_register/converter/company_converters/uk_company.py:131]: We have to remove the file and measure changes)

File: business_register/converter/company_converters/ukr_company.py (hunk 2 above)
Comment (IegorSoboliev) [reply business_register/converter/company_converters/ukr_company.py:802]: We have to remove the file and measure changes)

**Review by roman-tiukh — APPROVED** [review]

**Review by PyDev777 — APPROVED** [review]

**Pull request merged**